src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import java.util.Map;
+
+public interface OperationHandler {
+    Map<String, Integer> handle(String fruit, int quantity);
```

Comment:
I have seen our comment about returning map for tests, but I think we can just call and verify the storage directly like we did it last time, so Id still make it void